tests/integ/sagemaker/lineage/test_action.py

```diff
@@ -90,12 +94,14 @@ def test_tag(action_obj, sagemaker_session):
         )["Tags"]
         if actual_tags:
             break
+        time.sleep(1)
```

Comment:
probably can increase sleep? it's not going to save much time to check every 1s and by increasing to, say, 5s, you reduce call volumes (and potential throttling) by a factor 5.

Reply:
Good point, will update. Thank you